### DISCUSSION TITLE
How can i check what provider has been used?

### DISCUSSION THREAD

**Discussion by killerbossoriginal** [nextauthjs/next-auth#5702] [Help]
### Question 💬

How can i check what provider has been used?

### How to reproduce ☕️

idk

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by ThangHuuVu** [comment]
You can use Advance Initialization to inspect the request: https://next-auth.js.org/configuration/initialization#advanced-initialization

**Reply by killerbossoriginal** [reply]
But how can i do?
